lib/geocoder/opencagegeocoder.js

```diff
@@ -28,7 +28,29 @@ OpenCageGeocoder.prototype._geocode = function (value, callback) {
   var _this = this;
 
   var params = this._getCommonParams();
-  params.q = value;
+  if (value.q) {
+    if (value.bounds) {
+      if (Array.isArray(value.bounds)) {
+        params.bounds = value.bounds.join(',');
+      }
+      else {
+        params.bounds = value.bounds;
+      }
+    }
+    if (value.countrycode) {
+      params.countrycode = value.countrycode;
+    }
+    if (value.limit) {
+      params.limit = value.limit;
+    }
+    if (value.min_confidence) {
+      params.min_confidence = value.min_confidence;
+    }
+    params.q = value.q;
```

Comment:
GoogleGeocoder allow the same comportement but the q parameter is named `address` could you rename it to be coherent